dockstore-webservice/src/main/resources/migrations.1.10.0.xml

```diff
@@ -19,7 +19,7 @@
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd"
                    context="1.10.0">
     <changeSet author="dyuen" id="notification_seq">
-        <addAutoIncrement tableName="notification" columnName="id" startWith="100"/>
```

Comment:
Don't know how I feel about this, feels like historical revisionism

Reply:
Rolled it back, slipped in as an experiment.